Polynomials.Core/Polynomial.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Polynomials.Core
+{
+    //// https://stackoverflow.com/questions/3874627/floating-point-comparison-functions-for-c-sharp
```

Comment:
Please don't do that :D This is a fancy way of getting a bit values from float value, but in this task, please don't do that

Reply:
This won't help in a situation (0.1+0.1+0.1) == 0.3

Reply:
Yes I just left it here to let know I found it :D is gone now.